### DISCUSSION TITLE
[557600]Gorogoa画中世界 本体

### DISCUSSION THREAD

**Discussion by lueluo** [pjy612/SteamManifestCache#169] [Share]
`appid;557600`
keys:
```
557601;3dbed8d99e716a2c4481b39e1b841b09d8984a126afa6acb9f67ca8471289d62
```
code：AC90A6D3EDDB685C
email;jueluo12@163.com
[Gorogoa.zip](https://github.com/pjy612/SteamManifestCache/files/13767768/Gorogoa.zip)

**Comment by pjy612** [comment]
请调整格式
